psamm/database.py

```diff
@@ -195,8 +234,32 @@ def get_reaction_values(self, reaction_id):
             raise ValueError('Unknown reaction: {}'.format(repr(reaction_id)))
         return iteritems(self._reactions[reaction_id])
 
-    def get_compound_reactions(self, compound_id):
-        return iter(self._compound_reactions[compound_id])
+    def get_compound_reactions(self, compound):
+        return iter(self._compound_reactions[compound])
+
+    def set_model_name(self, model_name):
+        self._model_name = model_name
+
+    def set_extracellular_compartment(self, extracelluar_compartment):
+        self._extracellular_compartment = extracelluar_compartment
+
+    def set_default_flux_limit(self, flux_limit):
+        self._default_flux_limit = flux_limit
```

Comment:
Instead of these three methods, use the properties above to make these settable. See the documentation on `@property` on how to do this.